Riot/AppDelegate.m

```diff
@@ -1710,20 +1733,21 @@ - (void)addMatrixCallObserver
         [[NSNotificationCenter defaultCenter] removeObserver:matrixCallObserver];
     }
     
-    // Register call observer in order to handle new opened session
-    matrixCallObserver = [[NSNotificationCenter defaultCenter] addObserverForName:kMXCallManagerNewCall object:nil queue:[NSOperationQueue mainQueue] usingBlock:^(NSNotification *notif)
-    {
-        
+    void (^handleNotification)(NSNotification *) = ^(NSNotification *notif) {
```

Comment:
I did it because of readability reason. It was sometimes hard to work with this code on a small screen.